SPECS/azure-iotedge/0001-set-mgmt-socket-to-var-lib.patch

```diff
@@ -0,0 +1,19 @@
+This patch changes the default socket location in the iotedge management
+application to be in /var/lib/, the canonical location for non-debian Linux.
+
+This saves the user from having to specify --host unix:\\<path-to>.socket every
+time they use the iotedge management app.
+
+diff --git a/iotedge/src/main.rs b/iotedge/src/main.rs
+index b82f57d6..1f2f72a9 100644
+--- a/iotedge/src/main.rs
++++ b/iotedge/src/main.rs
+@@ -67,7 +67,7 @@ fn run() -> Result<(), Error> {
+             )
+         } else {
+             (
+-                Cow::Borrowed("unix:///var/run/iotedge/mgmt.sock"),
++                Cow::Borrowed("unix:///var/lib/iotedge/mgmt.sock"),
```

Comment:
This change may not be needed. See the fix for CentOS. If a fix is needed, it can be configured via an environment variable.

Reply:
It turns out the CentOS fix is somewhat different... CentOS 7 is old enough that systemd socket activation does not work quite the same as on more recent distros.

Mariner has a recent version of systemd so socket activation works fine, but this socket file happens to live under `/var/lib` rather than `/var/run`.